src/clib/xtg/surf_get_z_from_ij.c

```diff
@@ -29,9 +31,7 @@
  *    xori, yori    i      Map origins (real or relative)
  *    xinc, yinc    i      Map increments
  *    p_map_v       i      Pointer to map values to update
- *    flag          i      Flag for options
- *    debug         i      Debug flag
- *
+ *    option        i      0: interpolate, 1: nearest node
```

Comment:
I do not fully understand this. I regard "nearest node" as a form of interpolation.

Reply:
Nearest node is snapping value from nearest node

Reply:
Yes, but here it looks like "nearest node" it is something else than "interpolation", which it is not..